src/org/labkey/test/components/ui/entities/EntityInsertPanel.java

```diff
@@ -139,6 +147,13 @@ private EditableGrid grid()
         return new EditableGrid.EditableGridFinder(_driver).timeout(WAIT_FOR_JAVASCRIPT).waitFor(this);
     }
 
+    public EntityInsertPanel setUpdateDataForFileUpload(boolean checked)
```

Comment:
This checkbox appears on the file upload pane, and configures whether or not the file import process should overwrite or update existing records